src/Core/Utils.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+* Copyright (c) Microsoft Corporation.  All Rights Reserved.
+* Licensed under the MIT License.  See License in the project root
+* for license information.
+*
+* Utils File
+* PHP version 7
+*
+* @category  Library
+* @package   Microsoft.Graph
+* @copyright 2016 Microsoft Corporation
+* @license   https://opensource.org/licenses/MIT MIT License
+* @version   GIT: 0.1.0
+* @link      https://graph.microsoft.io/
+*/
+
+namespace Microsoft\Graph\Core;
+
+use GuzzleHttp\Exception\BadResponseException;
+use Microsoft\Graph\Exception\GraphException;
+
+/**
+ * Class Utils
+ *
+ * @category Library
+ * @package  Microsoft.Graph
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://graph.microsoft.io/
+ */
+class Utils
```

Comment:
Please call this something more specific, e.g. `ExceptionWrapper`.

As the code evolves, people dump code into these generic looking packages/classes and classes like `Utils` become very convoluted.

Related reading: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
Thanks for this suggestion & related reading @zengin . Made the change.